web/src/reload.js

```diff
@@ -99,6 +96,8 @@ function preserve_state(send_after_reload, save_pointer, save_narrow, save_compo
             if ($narrow_row.length > 0) {
                 url += "+narrow_offset=" + $narrow_row.get_offset_to_window().top;
             }
+        } else {
+            url += "+offset=" + message_lists.home.pre_narrow_offset;
```

Comment:
I guess this is preserving behavior, but does this make sense in the case that you're looking at the inbox/recent view? I wonder if this was a bug.

Reply:
Is there a specific bug you have in mind. We can file an issue for it and track it separately.